02-peer-protocol.md

```diff
@@ -776,6 +787,31 @@ satoshis, which is possible if `dust_limit_satoshis` is below 546 satoshis).
 No funds are at risk when that happens, but the channel must be force-closed as
 the closing transaction will likely never reach miners.
 
+### Closing Rejected: `closing_rejected`
+
+If the fundee disagrees on the `fee_range` the funder proposed, they may send a
+`closing_rejected` to tell them to try again.
+
+1. type: 40 (`closing_rejected`)
+2. data:
+   * [`channel_id`:`channel_id`]
```

Comment:
Comment from spec meeting: should this also include a fee range in this message?

Reply:
I thought we were going with the whole warnings approach for telling what parameters you didn't like about _any_ message?